src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  background-color: $backgroundAndBuyButtonOnHoverAndButtonTextColor;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &--layout {
```

Comment:
why do you need this class?
move     display: flex and flex-direction: column to card class